synthetics/test_schema.go

```diff
@@ -406,7 +406,7 @@ func makeTestTraceSchema(mode schemaMode) *schema.Schema {
 
 func makeUserInfoSchema() *schema.Schema {
 	return makeReadOnlyNestedObjectSchema(map[string]*schema.Schema{
-		"id": {
```

Comment:
Same as above

Reply:
Answered there

Reply:
I was about to ask the same question couple weeks ago and then I've answered myself :). The solution makes sense to me. It though may be useful to add a comment explaining this logic close to the code, so that future generations are less surprised.

Reply:
From my point of view, the developer can just jump to the definition of constant to see its value. IMO there is no need for additional comments.